.browserslistrc

```diff
@@ -0,0 +1,34 @@
+and_chr 115
```

Comment:
Do we need this file? I think you can just define this in package.json:
```
"browserslist": [
    "defaults and supports es6-module",
    "maintained node versions"
  ]
```

https://github.com/browserslist/browserslist#browserslist-

Reply:
Let me try it.

Reply:
And?

Reply:
@kilemensi It will only work if there is a `.browserslistrc` file, even an empty one. Adding in to `package.json` or deleting it caused starting the app to fail. Similar issue and fix [here](https://github.com/angular/angular-cli/issues/22606#issuecomment-1025206301)